back/app/models/app_configuration.rb

```diff
@@ -100,6 +100,11 @@ def feature_activated?(setting_name)
     settings[setting_name]&.values_at('enabled', 'allowed')&.all?
   end
 
+  def activate_feature!(setting_name)
```

Comment:
I introduced the same method in my PR 😅

Reply:
Just saw it haha! Great minds think alike